docs/ingestion/index.md

```diff
@@ -287,44 +289,31 @@ definition is an _ingestion spec_.
 
 Ingestion specs consists of three main components:
 
-- [`dataSchema`](#dataschema), which configures the [datasource name](#datasource), [input row parser](#parser),
-   [primary timestamp](#timestampspec), [flattening of nested data](#flattenspec) (if needed),
-   [dimensions](#dimensionsspec), [metrics](#metricsspec), and [transforms and filters](#transformspec) (if needed).
-- [`ioConfig`](#ioconfig), which tells Druid how to connect to the source system and . For more information, see the
+- [`dataSchema`](#dataschema), which configures the [datasource name](#datasource),
+   [primary timestamp](#timestampspec), [dimensions](#dimensionsspec), [metrics](#metricsspec), and [transforms and filters](#transformspec) (if needed).
+- [`ioConfig`](#ioconfig), which tells Druid how to connect to the source system and how to parse data. For more information, see the
    documentation for each [ingestion method](#ingestion-methods).
 - [`tuningConfig`](#tuningconfig), which controls various tuning parameters specific to each
   [ingestion method](#ingestion-methods).
 
-Example ingestion spec for task type "index" (native batch):
+Example ingestion spec for task type `parallel_index` (native batch):
 
 ```
 {
-  "type": "index",
+  "type": "parallel_index",
```

Comment:
this should be `index_parallel` - same comment on line 299, 332, 349. I have a doc change coming up so I can fix in the next patch as well.

Reply:
Oops, thanks. Fixed.